### ISSUE TITLE
im trying to run with mono  pythonnet

### ISSUE THREAD

**Issue by traderpedroso** [mathpaquette/IQFeed.CSharpApiClient#142]
``` IQFeedLauncher.Start("USER", "PASS", "PRODUCT ID")
System.ComponentModel.Win32Exception: Cannot find the specified file
  at System.Diagnostics.Process.StartWithShellExecuteEx ```

**Comment by mathpaquette** [comment]
Yes let me check how to handle that. Do you have a Pull request or fix already ?

Sent from my iPhone

On May 29, 2022, at 9:39 AM, Emerson Pedroso ***@***.***> wrote:

﻿

System.ComponentModel.Win32Exception: Cannot find the specified file
  at System.Diagnostics.Process.StartWithShellExecuteEx ```

**Comment by traderpedroso** [comment]
> Yes let me check how to handle that. Do you have a Pull request or fix already ?
> […](#)

acI have no pull request and could find the solution for that! thank you for your response

**Comment by mathpaquette** [comment]
@traderpedroso any progress ?

**Comment by BunkerCoder** [comment]
@mathpaquette  - Saw your question on #142.  Reproduce by removing the directory that contains the iqconnect.exe file from your system PATH. Usually C:\Program Files\DTN\IQFeed.
